### ISSUE TITLE
inconsistent field names

### ISSUE THREAD

**Issue by iroth** [contentful/contentful-management.py#18]
I have a field in my content model with id = "otherNames".
I am retrieving an entry and when I try to update this field by doing this:
setattr(entry, 'otherNames', newVal)
entry.save()
all seems to be OK, but no update happens.
When I debug the code, I see that the entry I received via the api has the field in the fields list under the en-US locale the name "other_names".
If I use setattr(entry, 'other_names', newVal), it will then work, but not if the entry had no value before the update. In that case I will get an exception that the fields is missing.
What's the right way?

**Comment by dlitvakb** [comment]
Hey @iroth,

Which version of the SDK are you using?

The latest version (1.3.1) - Fixes an issue when fields with camel cased IDs that weren't present on the original entry (due to the field being manually voided or never filled) are added to the entry.

If you're on a previous version, I'd recommend you to upgrade and try again.

As for what's the proper way to update the fields:

```python
entry = client.entries('space_id').find('entry_id')
entry.my_field = 'my value'
entry.save()
```

All the setters are the snake cased version of the field ID, the SDK converts it afterwards to the proper ID. If you want to look more into how this is done, the code responsible for that can be found here: https://github.com/contentful/contentful-management.py/blob/master/contentful_management/resource.py#L252RL255 and https://github.com/contentful/contentful-management.py/blob/master/contentful_management/entry.py#L89RL93

Hope that helps,

Cheers

**Comment by iroth** [comment]
Hi @dlitvakb - I think there is something wrong in the snake casing logic. When I look at the GUI, the id is definitely not camel cased. It appears as "otherNames".
I am using 1.3.1 which seems to improved the situation when updating entries that did not have fields populated before, but there is some issue with those double word names. I ended up creating a new field and just naming it with single word to bypass the issue

**Comment by dlitvakb** [comment]
Hey @iroth,

The logic in the SDK is there in order to simplify mapping to "pythonic" properties. As `camelCase` is not something you often see in Python.

As of 1.3.1, updates on entries should not be an issue when dealing with any ID.

Next release will also include a fix for Content Type updates with `camelCased` IDs.

Cheers

**Comment by dlitvakb** [comment]
This has already been fixed with the release of 1.3.2.

Cheers
